### PR TITLE
Add spock quickstart documentation

### PR DESCRIPTION
**Pull request by RSchlenker**
References: #1162 

Thanks @antrew & @kiview 
Created at hackergarten @ gr8conf 2019.

### CODE REVIEW

**Comment by kiview** [comment]
Of course there is quite some content overlap with the JUnit4 quickstart documentation.
We decided to do this, to give Spock users the complete information necessary without looking at the JUnit4 quickstart.

Is it possible to include text snippets from other markdown files?

**Comment by rnorth** [comment]
This looks great, thanks! I think it's also conceptually close to @bmuschko's #1416 - it's good to move in the same direction.

Re:
> Is it possible to include text snippets from other markdown files?

Could you clarify further? Are you thinking about reducing duplication between the `*_quickstart.md` files?

**Comment by kiview** [comment]
> Could you clarify further? Are you thinking about reducing duplication between the *_quickstart.md files?

Exactly this. So the introductory text is the same, but it makes sense to have everything right there on this single page, it will give a new user the best quickstart experience.

File: docs/quickstart/spock_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    <groupId>org.testcontainers</groupId>
+    <artifactId>spock</artifactId>
+    <version>{{latest_version}}</version>
+    <scope>test</scope>
+</dependency>
+```
+
+## 2. Get Testcontainers to run a Redis container during our tests
+
+Annotate the test class with Testcontainers specification:
```
Comment (kiview) [inline docs/quickstart/spock_quickstart.md]:
```suggestion
Annotate the Spock specification class with the Testcontainers extension:
```

**Comment by rnorth** [comment]
I don't think we can do straight includes in markdown at present, but that's maybe a future improvement we can make.

**Comment by kiview** [comment]
Merged, thanks a lot @RSchlenker and @antrew!

**Pull request merged**